torch/_export/passes/fake_tensor_prop.py

```diff
@@ -0,0 +1,18 @@
+from torch.fx.interpreter import Interpreter
+
+
+class FakeTensorProp(Interpreter):
+    def run(self):
+        inp = tuple(
+            node.meta["val"]
+            for node in self.module.graph.nodes
+            if node.op == "placeholder"
+        )
```

Comment:
Curious whether the placeholders are guaranteed to be in the original input order among the graph nodes?

Reply:
yes, placeholders should be in the same order as the original input.